.travis.yml

```diff
@@ -109,12 +109,31 @@ jobs:
           branch: master
           repo: denoland/deno
         skip-cleanup: true
+    
+    - name: "cargo release linux arm64"
+      os: linux
+      dist: xenial
+      script:
+      - rustup target add aarch64-unknown-linux-gnu
+      - sudo apt update
+      - sudo apt -yq --no-install-suggests --no-install-recommends install g++-5-aarch64-linux-gnu gcc-5-aarch64-linux-gnu g++-5-multilib libc6-arm64-cross
+      - build/linux/sysroot_scripts/install-sysroot.py --arch=arm64
+      - export DENO_BUILD_ARGS="target_cpu=\"arm64\" v8_target_cpu=\"arm64\""
+      - export CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER="/usr/bin/aarch64-linux-gnu-gcc-5"
+      - export CC_aarch64_unknown_linux_gnu="/usr/bin/aarch64-linux-gnu-gcc-5"
+      - cargo build -vv --target=aarch64-unknown-linux-gnu --release --locked
+      - cargo build -vv --target=aarch64-unknown-linux-gnu --release --tests --locked
+      - mv $(find $CARGO_TARGET_DIR/aarch64-unknown-linux-gnu/release -maxdepth 1 -type f -executable -regextype sed -regex '.*/[a-z_]*-[0-9a-f]*') $CARGO_TARGET_DIR/aarch64-unknown-linux-gnu/release/test_rs
+      - sudo apt -yq install qemu qemu-user binfmt-support qemu-user-binfmt
+      - sudo ln -s /usr/aarch64-linux-gnu/lib/ld-linux-aarch64.so.1 /lib/ld-linux-aarch64.so.1
+      - export QEMU_LD_PREFIX=/usr/aarch64-linux-gnu
+      - $CARGO_TARGET_DIR/aarch64-unknown-linux-gnu/release/deno tests/002_hello.ts
+      # - DENO_BUILD_MODE=release ./tools/test.py $CARGO_TARGET_DIR/aarch64-unknown-linux-gnu/release TODO(afinch7): Get the tests working
```

Comment:
tools/test.py requires test_cc to be built as well - which isn't built thru cargo... so we'll have to figure this out. 

But great work! This is a huge first step and I'm very happy to have this in CI.

Reply:
I added it into `deno_deps_cross` group, but it's currently throwing a error that looks really familiar.
```
#
# Fatal error in , line 0
# unreachable code
#
#
#
#FailureMessage Object: 0x4001224930
==== C stack trace ===============================

    target/aarch64-unknown-linux-gnu/release/test_cc(+0x9da184) [0x40009da184]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x9d9750) [0x40009d9750]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x9d580c) [0x40009d580c]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x7f61bc) [0x40007f61bc]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x6c44dc) [0x40006c44dc]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x74b558) [0x400074b558]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x74b4a0) [0x400074b4a0]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x74a08c) [0x400074a08c]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x74b328) [0x400074b328]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x749588) [0x4000749588]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x21be90) [0x400021be90]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x754718) [0x4000754718]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x600548) [0x4000600548]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x75335c) [0x400075335c]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x1dc7d4) [0x40001dc7d4]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x1dc8bc) [0x40001dc8bc]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x1c5e80) [0x40001c5e80]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x1b6dc8) [0x40001b6dc8]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x983bac) [0x4000983bac]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x9844e0) [0x40009844e0]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x984944) [0x4000984944]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x990238) [0x4000990238]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x98fcec) [0x400098fcec]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x1c53fc) [0x40001c53fc]
    /lib/libc.so.6(__libc_start_main+0xe0) [0x40013ab6e0]
    target/aarch64-unknown-linux-gnu/release/test_cc(+0x1b3034) [0x40001b3034]
``` 
Almost certain this is the same initial issue I was dealing with on arm builds. 
Once we have a fix for snapshot generation for cross compiled builds this should be fixed.